ale/base/label_isis.py

```diff
@@ -170,14 +170,22 @@ def spacecraft_clock_start_count(self):
         : str
           Spacecraft clock start count
         """
-        if 'SpacecraftClockStartCount' in self.label['IsisCube']['Instrument']:
-            return str(self.label['IsisCube']['Instrument']['SpacecraftClockStartCount'])
-        elif 'SpacecraftClockCount' in self.label['IsisCube']['Instrument']:
-            return str(self.label['IsisCube']['Instrument']['SpacecraftClockCount'])
-        elif 'SpacecraftClockStartCount' in self.label['IsisCube']['Archive']:
-            return str(self.label['IsisCube']['Archive']['SpacecraftClockStartCount'])
-        else:
-            return None
+        if not hasattr(self, "_clock_start_count"):
+            if 'SpacecraftClockStartCount' in self.label['IsisCube']['Instrument']:
+                self._clock_start_count = self.label['IsisCube']['Instrument']['SpacecraftClockStartCount']
+            elif 'SpacecraftClockCount' in self.label['IsisCube']['Instrument']:
+                self._clock_start_count = self.label['IsisCube']['Instrument']['SpacecraftClockCount']
+            elif 'SpacecraftClockStartCount' in self.label['IsisCube']['Archive']:
+                self._clock_start_count = self.label['IsisCube']['Instrument']['SpacecraftClockStartCount']
+            else:
+                self._clock_start_count = None
+
+            if isinstance(self._clock_start_count, pvl.Quantity):
```

Comment:
Is this because of a potential PVL version change or can we get back different types?

Reply:
PvL returns different types depending on if it's just a value or a value with units.

Reply:
Note the difference between:

```
self._clock_start_count
```
and
```
self._clock_start_count.value
```